src/input_context/bind.rs

```diff
@@ -0,0 +1,172 @@
+use crate::prelude::*;
+
+#[derive(Debug, Default)]
+pub struct BindConfigInfo {
+    pub modifiers: Vec<Box<dyn InputModifier>>,
+    pub conditions: Vec<Box<dyn InputCondition>>,
+}
+
+#[derive(Debug, Default)]
+pub struct BindConfigSet {
+    pub binds: Vec<BindConfigs>,
+    pub info: BindConfigInfo,
+}
+
+impl FromIterator<BindConfigs> for BindConfigSet {
+    fn from_iter<T: IntoIterator<Item = BindConfigs>>(iter: T) -> Self {
+        Self {
+            binds: iter.into_iter().collect(),
+            info: BindConfigInfo::default(),
+        }
+    }
+}
+
+impl Extend<BindConfigs> for BindConfigSet {
+    fn extend<T: IntoIterator<Item = BindConfigs>>(&mut self, iter: T) {
+        self.binds.extend(iter)
+    }
+}
+
+#[derive(Debug)]
+pub struct BindConfig {
```

Comment:
Since we removing `InputBind`, maybe re-use it name here? :thinking: I.e. InputBindSet, InputBinds, etc.

Just not a fan of the word "config" in this case.

Reply:
agreed, renamed